docs/rest_api.rst

```diff
@@ -663,3 +663,51 @@ be rejected.
   :statuscode 404: If the channel does not exist
   :statuscode 409: If the given block number argument is invalid
   :statuscode 500: Internal Raiden node error
+  
+
+.. http:get:: /api/1/channels/history/(token_address)/(partner_address)
+
+   Query for all internal events tied to a specific token network channel. If the partner is not provided, the client will provide a list containing all events for all channels.
+
+   **Example Request**:
+
+   .. http:example:: curl wget httpie python-requests
+
+      GET /channels/history/0x0f114A1E9Db192502E7856309cc899952b3db1ED/0x82641569b2062B545431cF6D7F0A418582865ba7 HTTP/1.1
+      Host: localhost:5001
+
+  **Example Response**:
+
+  .. sourcecode:: http
+
+     HTTP/1.1 200 OK
+     Content-Type: application/json
+
+    [
+        {
+            block_number: 3694255,
+            event: "EventTransferReceivedSuccess",
+            identifier: 1532342045500,
+            amount: 5,
+            initiator: "0x82641569b2062B545431cF6D7F0A418582865ba7"
+        },
+        {
+            block_number: 3694245,
+            event: "EventTransferSentSuccess",
+            identifier: 1532341917127,
+            amount: 35,
+            target: "0x82641569b2062B545431cF6D7F0A418582865ba7"
+        },
+        {
+            block_number: 3694216,
+            event: "EventTransferSentSuccess",
+            identifier: 1532341606278,
```

Comment:
We need to decide if we should output the `identifier`. @LefterisJP @hackaugusto

Reply:
Is this the database identifier? It shouldn't be exposed IMO